pages/press/index.js

```diff
@@ -0,0 +1,365 @@
+                <TextLink href="https://login.salesforce.com/?locale=es" target="_blank" className="no-underline">
+                  <b>Salesforce</b>
+                </TextLink>: Business consultancy company. It provides (CRM) service and other services. 
+                It manages from small businesses to FORTUNE 500 companies and it has more than 150.000 customers.
+              </Paragraph>
+            </li>
+            <li className="py-1">
+              <Paragraph typeStyle="body-md" className="my-4">
+                <TextLink href="https://www.ibm.com" target="_blank" className="no-underline">
+                  <b>IBM</b>
+                </TextLink>: Company that manufactures and markets hardware and software. 
+                It has operations in over 170 countries and provides hosting and consulting services in many areas.
+              </Paragraph>
+            </li>
+            <li className="py-1">
+              <Paragraph typeStyle="body-md" className="my-4">
+                <TextLink href="https://www.sap.com/index.html" target="_blank" className="no-underline">
+                  <b>SAP</b>
+                </TextLink>: Company dedicated to the design of computer products for business management. 
+                Develops business software to manage operations and business-to-customer relationships. 
+                It’s a large company with 100,330 employees.
+              </Paragraph>
+            </li>
+            <li className="py-1">
+              <Paragraph typeStyle="body-md" className="my-4">
+                <TextLink href="https://www.iqvia.com/" target="_blank" className="no-underline">
+                  <b>IQVIA</b>
+                </TextLink>: Company providing services for the combined health information technology and clinical research industries. 
+                It employs more than 58.000 people in over 100 countries
+              </Paragraph>
+            </li>
+          </ul>
+        </Paragraph>
+
+        <Heading level="h3" typeStyle="heading-md" className="my-5">
+          <span id="funding">Funding</span>
+        </Heading>
+        <Paragraph typeStyle="body-md" className="my-4">
+          AsyncAPI is an open source project and is therefore funded by talent and voluntary contributions. 
+          It relies on the expertise and dedication of the <b>Community developers</b> who believe in the project and contribute to its evolution.
+        </Paragraph>
+        <Paragraph typeStyle="body-md" className="my-4">
+          Financially, there are individual donations and contributions made by users of the project, and others that come from different companies and initiatives that trust and support AsyncAPI through sponsorship. 
+          The main sponsors that contribute to sustain this initiative are Postman, IQVIA Technology, Mulesoft, Salesforce, SAP, SLACK, Solace, and Tibco.
+        </Paragraph>
+        <Paragraph typeStyle="body-md" className="my-4">
```

Comment:
It doesn't render properly in the UI. I think because of `[` and `]`, just remove those. And also only link to https://opencollective.com/asyncapi because we do not really get donations over GitHub sponsors, almost nothing